libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -317,47 +312,31 @@ impl<'a> JsiiImporter<'a> {
 	}
 
 	/// Import a JSII interface into the Wing type system.
-	/// In JSII, a struct is a special kind of interface whose name starts with "I" which has no methods and whose
-	/// properties are all readonly.
+	///
+	/// In JSII an interface can either be a "struct" (for data types) or a "behavioral" interface (for normal
+	/// interface types).
+	/// A struct's name always starts with "I", it has no methods, and its properties are all readonly.
```

Comment:
It's validated by the jsii compiler which is why we don't need to validate it on import. I think it's OK to panic on invalid JSII libraries since this shouldn't happen in practice (famous last words?)